package.json

```diff
@@ -19,8 +19,8 @@
     "coverage": "istanbul cover _mocha -- -t 20000 test/index.js",
     "coveralls": "npm run coverage && coveralls < coverage/lcov.info",
     "standard": "standard",
-    "test": "mocha --reporter list -t 20000 test/index.js",
-    "bundle-test": "browserify test/index.js > test/bundle.js"
+    "test": "mocha --reporter spec -t 20000 test/index.js",
+    "test:bundle": "browserify test/index.js >test/browser.js"
```

Comment:
well not karma because we aren't doing karma there, we don't tend to use colons to separate out related tasks so that might be a bit confusing why don't we change that back and then open a separate issue to change it